brainscore/metrics/xarray_utils.py

```diff
@@ -88,7 +88,7 @@ def __call__(self, prediction, target):
             target_neuroids = target.isel(**{neuroid_dims[0]: i})  # `isel` is about 10x faster than `sel`
             prediction_neuroids = prediction.isel(**{neuroid_dims[0]: i})
             r, p = self._correlation(target_neuroids, prediction_neuroids)
-            correlations.append(r)
+            correlations.append(r if np.isfinite(r) else 0)
```

Comment:
let's just stick with `r`. If the correlation is `nan`, then that's what it is at this point, not 0.

Reply:
When one of the variable is constant, pearsonr gives nan. Mathematically, it should be zero as they are uncorrelated. This can happen a lot in the single neuron regression mapping when the units do not respond to the stimuli.

Reply:
I would really prefer to stick with the standard scipy output and only fiddle with it when needed, i.e. in this case in the single neuron regression (where we could drop `nan`s or set them to `0`).